tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "esModuleInterop": true,
     "importHelpers": true,
     // See https://devblogs.microsoft.com/typescript/typescript-and-babel-7/
+    "noEmit": true,
```

Comment:
Why this change?

Reply:
I would have to clone and reinstall the repo to be sure, but I was having issues with babel/typescript interop, this made it so that the Babel transforms worked (in my environment at least). I added this based on MS's own ts/babel config setup:
![Screen Shot 2021-09-28 at 2 36 33 PM](https://user-images.githubusercontent.com/87667330/135145978-d15d09f8-917e-4da6-b98c-ddd864d47fd2.png)

Reply:
Same here, revert this please